static/css/style.css

```diff
@@ -333,7 +333,6 @@ html[data-useragent*='MSIE 10.0'] .small-icons .status-icon {
 
 .roundlabel {
   padding-top: .3rem;
-  padding-bottom: .3rem;
```

Comment:
When testing this in all browser, could we take screenshots as well?

Reply:
Chrome Catalina
![Screenshot 2019-12-11 at 13 57 52](https://user-images.githubusercontent.com/31242877/70627586-66f2b480-1c1e-11ea-8ee4-bdd1c371d4d1.png)
![Screenshot 2019-12-11 at 13 57 44](https://user-images.githubusercontent.com/31242877/70627587-66f2b480-1c1e-11ea-9b46-bdbdb8ba5230.png)
Chrome Win10
![Screenshot 2019-12-11 at 13 57 07](https://user-images.githubusercontent.com/31242877/70627589-66f2b480-1c1e-11ea-9e2a-93f43551dc08.png)
![Screenshot 2019-12-11 at 13 56 56](https://user-images.githubusercontent.com/31242877/70627590-66f2b480-1c1e-11ea-8bae-ec497e38067d.png)
IE11 Win10
![Screenshot 2019-12-11 at 13 56 28](https://user-images.githubusercontent.com/31242877/70627591-678b4b00-1c1e-11ea-9fa5-cdf44fa0f84d.png)
![Screenshot 2019-12-11 at 13 56 21](https://user-images.githubusercontent.com/31242877/70627592-678b4b00-1c1e-11ea-8390-c24954a3b2c5.png)
Safari Catalina
![Screenshot 2019-12-11 at 13 54 55](https://user-images.githubusercontent.com/31242877/70627593-678b4b00-1c1e-11ea-8d9c-d980b72b446e.png)
![Screenshot 2019-12-11 at 13 54 48](https://user-images.githubusercontent.com/31242877/70627595-678b4b00-1c1e-11ea-8bb1-f861cb17b55f.png)

Reply:
Great, thanks Mark!
Could you do a quick check in English in IE11 and Firefox Windows? Just to make sure the button height isn't somehow reduced without the additional padding.

Reply:
FF
![Screenshot 2019-12-12 at 09 29 02](https://user-images.githubusercontent.com/31242877/70702330-cfe13780-1cc5-11ea-930c-f7beebb883f6.png)
IE11
![Screenshot 2019-12-12 at 09 28 38](https://user-images.githubusercontent.com/31242877/70702332-d079ce00-1cc5-11ea-92a7-8401dddf88dd.png)

Reply:
Right so it looks like the padding is needed to keep the height to the buttons and the blue triangles corner aligned:

master:
![image](https://user-images.githubusercontent.com/29712657/70703260-9d383e80-1cc7-11ea-856b-5eb20677d5ff.png)

This branch:
![image](https://user-images.githubusercontent.com/29712657/70703332-ca84ec80-1cc7-11ea-96ee-bc9c7157cdf3.png)

We need to find a way to keep the same button height independently of the language.

Reply:
I've set max width and height of the buttons in https://github.com/bbcmicrobit/PythonEditor/pull/306/commits/66b79eb2dc94ae1296fdc27976f42c60f2558284 this has the affect of cutting off long label translations
<img width="139" alt="Screenshot 2019-12-16 at 08 01 19" src="https://user-images.githubusercontent.com/31242877/70889333-43db5280-1fda-11ea-842e-fb7b3bce331a.png">
<img width="122" alt="Screenshot 2019-12-16 at 08 01 01" src="https://user-images.githubusercontent.com/31242877/70889334-43db5280-1fda-11ea-9992-b8ec2d9d5357.png">
but does resolve the issue of malformed buttons and not editing the CSS each time we add a non-latin language.

Reply:
Thanks Mark!
Can we ask the Croatian translators if there is a shorter way to translate that button?

Reply:
Will do

Reply:
See https://github.com/bbcmicrobit/PythonEditor/pull/306#issuecomment-572073475